CONTRIBUTING.md

```diff
@@ -34,9 +35,29 @@ If you submitted a Pull Request that fixes a ticket with the "\$100" tag, then y
 You can request a new feature by submitting a ticket to our [GitHub issues](https://github.com/jhipster/generator-jhipster/issues). If you
 would like to implement a new feature then consider what kind of change it is:
 
--   **Major Changes** that you wish to contribute to the project should be discussed first. Please open a ticket which clearly states that it is a feature request in the title and explain clearly what you want to achieve in the description, and the JHipster team will discuss with you what should be done in that ticket. You can then start working on a Pull Request.
+-   **Major Changes** that you wish to contribute to the project should be discussed first. Please open a ticket which clearly states that it is a feature request in the title and explain clearly what you want to achieve in the description, and the JHipster team will discuss with you what should be done in that ticket. You can then start working on a Pull Request. In order to communicate major changes proposals and receive reviews from the core team, you can also submit an RFC.
 -   **Small Changes** can be proposed without any discussion. Open up a ticket which clearly states that it is a feature request in the title. Explain your change in the description, and you can propose a Pull Request straight away.
 
+## <a name="rfcs"></a> RFCs
+
+Sometimes, major feature requests are "complex" or "substantial". In this case, Github Issues might not be the best tool to to present them because we will need a lot of going back and forth to reach a consensus.
+
+So we ask that these feature request be put through a formal design process and have their specifications described in an "RFC" (request for comments) that will be validated by the team through a Pull Request Review.
+
+The RFC process is intended to provide a consistent and controlled path for major features and directions of the project.
+
+To submit an RFC follow those steps:
+
+1. Discuss the RFC proposal with the core team through Github issues or other channels
+2. Create the initial Github issue for the Feature Request if it doesn't already exist
+3. Copy the `rfcs/0-jhipster-rfc-template.md` to `rfcs/${featureRequestIssueNumber}-my-feature-request-name.md`
+4. Fill in the RFC, make sure to complete every required section
+5. Submit the RFC as a Pull Request
```

Comment:
May be, Add a step saying the a Gist of the proposal to be added as PR description??

Reply:
OK, I added this.